website/source/docs/install/index.html.md

```diff
@@ -9,19 +9,55 @@ description: |-
 
 # Install Consul
 
-Installing Consul is simple. There are two approaches to installing Consul:
+Installing Consul is simple. There are two main approaches to installing Consul:
 
-1. Using a [precompiled binary](#precompiled-binaries)
+1. [Installing on Kubernetes](#install-on-kubernetes) 
 
-1. Installing [from source](#compiling-from-source)
+1. Installing as a standalone binary 
```

Comment:
Could you add the link back? `(#precompiled-binaries)`

Reply:
This is included as a sub bullet link below.